solar/cell.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, area, temperature):
         self.area = area
 
         # efficiency of solar energy conversion to electrical energy
-        self.efficiency = .237
+        self.efficiency = .243
 
         # temp_coeff is the percentage decrease
         # in the cells' efficiency for every degree above 25C
```

Comment:
So I'm noticing that our efficiency is a bit weird. Right now, if our temperature is 26 (as an example), our efficiency calculations is

efficiency = (self.efficiency - self.temp_coefficient * (self.temperature - 25))
efficiency = (0.243 - (-0.0029) * (26 - 25))
efficiency = 0.2459

Which would tell us our efficiency goes up - that's counterintuitive. Is there something going wrong?
(sorry, this is an error already in the file, but this is a good chance to correct it)